secv_guis/bimask_app/main_window.py

```diff
@@ -294,7 +294,8 @@ def preannot_from_path(self, preannot_path, rgba, keep_p_value=0.05,
         """
         assert self.scene().img_pmi is not None, \
             "You need to load an image first!"
-        self._preannot_pmap = np.load(preannot_path)["entropy"]
+        im = Image.open(preannot_path)
+        self._preannot_pmap = np.asarray(im)#np.load(,allow_pickle=True)["entropy"]
```

Comment:
linting: double space before inline comment and single space after `#`. Anyway you can delete old code if not used anymore :)

Also: update docstring with new behaviour.
What kind of datatypes will Image handle? will this work for uint8, float32...? if you e.g. expect always floats, cast the array with `.astype(np.float32)`. If you expect any specific dtype, update the docstring with that information

Reply:
I see that you removed the old code in further commit, nevermind that

Reply:
fixed later